Makefile

```diff
@@ -1,4 +1,4 @@
-  
+
```

Comment:
How does this blank line that I keep removing keep getting added back to the Makefile?